src/shared/features/AppletSettings/ExportDataSetting/ExportDataSetting.tsx

```diff
@@ -1,167 +1,38 @@
+  onExportSettingsClose,
+}: ExportDataSettingProps) => {
   const { result: appletData } = applet.useAppletData() ?? {};
   const [dataIsExporting, setDataIsExporting] = useState(false);
-  const dataTestid = 'applet-settings-export-data-export';
-  const minDate = new Date(appletData?.createdAt ?? '');
-  const getMaxDate = () => getNormalizedTimezoneDate(new Date().toString());
-  const methods = useForm<ExportDataFormValues>({
-    resolver: yupResolver(exportDataSettingSchema() as ObjectSchema<ExportDataFormValues>),
-    defaultValues: {
-      dateType: ExportDateType.AllTime,
-      fromDate: minDate,
-      toDate: getMaxDate(),
-    },
-    mode: 'onSubmit',
-  });
-  const { control, setValue, watch } = methods;
-  const dateType = watch('dateType');
-  const fromDate = watch('fromDate');
-  const toDate = watch('toDate');
-  const hasCustomDate = dateType === ExportDateType.ChooseDates;
-  const commonProps = {
-    maxDate: getMaxDate(),
-    control,
-    inputSx: {
-      width: '19rem',
-      '& .MuiInputLabel-outlined': {
-        textTransform: 'capitalize',
-      },
-    },
-  };
-
-  const onFromDateSubmit = (date: DateType) => {
-    if (!date) return;
-    setValue('fromDate', startOfDay(date));
-  };
-  const onToDateSubmit = (date: DateType) => {
-    if (!date) return;
-    setValue('toDate', endOfDay(date));
-  };
-  const onCloseCallback = () => {
-    if (toDate < fromDate) {
-      setValue('toDate', addDays(fromDate, 1));
-    }
-  };
-  const onDateTypeChange = (e: SelectEvent) => {
-    const dateType = e.target.value as ExportDateType;
-    const maxDate = getMaxDate();
-    switch (dateType) {
-      case ExportDateType.AllTime:
-        setValue('fromDate', minDate);
-        setValue('toDate', maxDate);
-        break;
-      case ExportDateType.Last24h:
-        setValue('fromDate', addDays(maxDate, -1));
-        setValue('toDate', maxDate);
-        break;
-      case ExportDateType.LastWeek:
-        setValue('fromDate', addDays(maxDate, -7));
-        setValue('toDate', maxDate);
-        break;
-      case ExportDateType.LastMonth:
-        setValue('fromDate', addDays(maxDate, -30));
-        setValue('toDate', maxDate);
-        break;
-      case ExportDateType.ChooseDates:
-        setValue('fromDate', minDate);
-        setValue('toDate', maxDate);
-        break;
-    }
-  };
 
   return (
-    <FormProvider {...methods}>
-      <form noValidate autoComplete="off">
-        <StyledAppletSettingsDescription>{t('exportDescription')}</StyledAppletSettingsDescription>
-        <SelectController
-          name={'dateType'}
-          control={control}
-          options={getDateTypeOptions()}
-          label={t('dateRange')}
-          data-testid={`${dataTestid}-dateType`}
-          dropdownStyles={{
-            width: '15rem',
-          }}
-          SelectProps={{
-            autoWidth: true,
-          }}
-          customChange={onDateTypeChange}
+    <>
+      <ExportSettingsPopup
```

Comment:
```suggestion
     {isExportSettingsOpen && <ExportSettingsPopup
```
I suggest adding this check to prevent rendering the whole component if the modal is not open.